src/test/java/core/basesyntax/service/impl/WriteServiceImplTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.FruitTransactionService;
+import core.basesyntax.service.ReadService;
+import core.basesyntax.service.WriteService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class WriteServiceImplTest {
+    private final Path invalidPath = Path.of("src/test/java/resources/invalidFile.csv");
+    private final Path inputPath = Path.of("src/test/java/resources/inputTestFile.csv");
+    private final Path repotPath = Path.of("src/test/java/resources/reportTestFile.csv");
+    private final FruitTransactionService fruitTransactionService =
+            new FruitTransactionServiceImpl();
+    private final ReadService readService = new ReadServiceImpl();
+    private final WriteService writeService = new WriteServiceImpl();
+    

```

Comment:
Please add an 'BeforeEach' method to initialize the Storage.fruitShopData before each test.